index.html

```diff
@@ -638,7 +638,7 @@ <h3>Information Model</h3>
                     the <a>Thing Description Directory</a> can contain zero or more <a>TD</a>s.
                     For every TD, the directory additionally maintains the registration
                     information for bookkeeping and search purposes.
-                    A TD which embeds the registration information is called an <a>Enriched TD</a>.
+                    <a>Enriched TD</a>s embed the registration information inside them.
```

Comment:
There could be other kinds of metadata embedded in a TD besides registration information.  I would generalize this to say "metadata, including registration information"